### DISCUSSION TITLE
Hi tsyuay, maybe you're playing that game I wanted you to add. it's a very good game. pls!!! Last wish.

### DISCUSSION THREAD

**Discussion by tnt3333334** [Tsyuay/FrivReborn#40] [Q&A] [answered]
the name of the game is ballistix pro.
<img width="862" alt="ballistix pro" src="https://user-images.githubusercontent.com/122381404/213914319-07297243-192d-41ef-ae0c-90dc8c900641.png">

swf the game:https://web.archive.org/web/20150812204235/http://l.funo.com/z/ballistixpro/ballistixpro.swf

walkthrough:https://web.archive.org/web/20201230235714/https://www.friv.com/x/walkthroughs/ballistixpro.html

follow the links to put the game and the walkthrough.

**Comment by Tsyuay** [comment, marked as answer]
done
